### DISCUSSION TITLE
"ecmaVersion": 2022 gives me an error

### DISCUSSION THREAD

**Discussion by BrennerSpear** [eslint/eslint#15201] [Q&A]
```    "devDependencies": {
        "eslint": "^8.0.1",
        "eslint-config-prettier": "^8.3.0",
```

```
{
    "extends": ["eslint:recommended", "prettier"],
    "parserOptions": {
        "ecmaVersion": 2022,
        "sourceType": "module"
    },
    "env": {
        "node": true
    }
}

```

gives me `Value is not accepted. Valid values: 3, 5, 6, 2015, 7, 2016, 8, 2017, 9, 2018, 10, 2019, 11, 2020, 12, 2021, "latest".`

What do I have set up wrong that's giving me that error?

**Comment by nzakas** [comment]
Can you run `npx eslint -v` and check that you’re running the latest version?

**Comment by BrennerSpear** [comment]
```
~/r/birthblock-event-forwarder (main|✔) $ npx eslint -v                                                                                                                                           
v8.0.1
```

**Reply by aladdin-add** [reply]
ecma version 2022 should have been supported in eslint 8.0, so it looks something went wrong.

what's the output running `npm ls eslint espree`?  (and better to have a repro :)

**Reply by BrennerSpear** [reply]
looks like it's gone now... idk why it was giving me an issue yesterday!
